modules/web/src/app/core/services/cluster.ts

```diff
@@ -137,9 +138,12 @@ export class ClusterService {
     return this._http.post<Cluster>(url, createClusterModel);
   }
 
-  patch(projectID: string, clusterID: string, patch: ClusterPatch): Observable<Cluster> {
+  patch(projectID: string, clusterID: string, patch: ClusterPatch, skipKubeletValidation = false): Observable<Cluster> {
     const url = `${this._newRestRoot}/projects/${projectID}/clusters/${clusterID}`;
-    return this._http.patch<Cluster>(url, patch);
+    if (skipKubeletValidation) {
+      this._params = this._params.set('skip_kubelet_version_validation', skipKubeletValidation);
```

Comment:
I think `params` should be a local variable in this method, unless we want to save this value for future requests which don't require `skipKubeletValidation`?

Reply:
Yeah, that makes more sense. I was inspired by the way this class manages headers, but keeping it in the method's scope is a good idea.